phpcs.xml

```diff
@@ -150,6 +150,7 @@
 				@slowThreshold,
 				@noinspection
 			"/>
+            <property name="enableVoidTypeHint" value="false" />
```

Comment:
I'm okay with most everything we decide as a team - consciously and transparently.

Reply:
I thought you said I could remove this if it bugged me. That was some time ago though, so perhaps my memory on it is foobar

Reply:
@KaiNissen @gbirke Void return type hints are now not enforced any more.